lib/command-map.js

```diff
@@ -331,4 +342,4 @@ const commandMap = {
   }
 }
 
-module.exports = { commandMap }
+module.exports = { commandMap }
```

Comment:
Please add newline

Reply:
for some reason (bein dumb???) i do not know what exactly you mean

Reply:
oh it shows up in expanded view: 
![Screen Shot 2020-09-22 at 12 39 39 PM](https://user-images.githubusercontent.com/8183738/93911681-b3504200-fcd0-11ea-80ea-a773f2ede7a8.png)
Put a newline after the last line. It is a GitHub standard